plugins/org.yakindu.sct.doc.user/src/user-guide/statechart_language.textile

```diff
@@ -916,7 +925,16 @@ p(#fig_region-priorities).
 
 p=. Region priorities
 
-Not being able to raise an event and process it in "earlier" regions is a restriction you can overcome using variables. The idea is to set a "communication variable" to a certain value in a region that is processed "later". During the next RTC the earlier region could act on the variables value.
+Not being able to raise an event and process it in "earlier" regions is a restriction you can overcome using variables. The idea is to set a "communication variable" to a certain value in a region that is processed "later". During the next RTC the earlier region could act on the variables value. Another option – and in many cases the better on, is to change to the event-driven execution scheme.
```

Comment:
I have the feeling I've read this section three times now. Wouldn't it be more useful to use links and only explain it only once? At the very least, all repetitions have to be maintained.